Core/Config.h

```diff
@@ -270,7 +276,7 @@ struct Config {
 
 	// UI
 	bool bShowDebuggerOnLoad;
-	int iShowFPSCounter;
+	int iShowFPSCounter; // Should we rename this now?
```

Comment:
int iShowStatusFlags maybe? what do you think?

Reply:
Good to me

Reply:
Addressed in https://github.com/hrydgard/ppsspp/pull/16851/commits/ca154879f05e829c6c333c8f27c1ea0f1f65d4d2